infrastructure/output.tf

```diff
@@ -1,9 +1,9 @@
 output "vaultUri" {
-  value = "${module.key-vault.key_vault_uri}"
+  value = "${local.vaultUri}"
```

Comment:
I guess you could just refer straight to vault's output vault_uri param and get rid of this local one.